CHANGELOG.md

```diff
@@ -234,7 +234,7 @@
 ### 1.7.3 (2016 May 20)
 
 * [9b50305] v1.7.3
-* [3f641bd] Merge branch 'master' of github.com:bitrise-io/steps-xcode-archive
+* [3f641bd] Merge branch 'master' of github.com:bitrise-steplib/steps-xcode-archive
```

Comment:
instead pls remove the changelog as well, check the `Updated release flow` section of the repo revision guide

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/commits/78a85e677b3e813fe5420c2aaa334e1030510ebd